### PR TITLE
Encode OGC Modspec

### PR DESCRIPTION
**Pull request by anermina**
Notes:
- Red text from the original document is highlighted in the generated one.
- Smart terms were boldfaced in the original document. They are italicized in the generated one, i.e. usual markup for the smart terms is used. However, some other terms are boldfaced in the original document (and I kept the original formatting), so we now have a mixture of italicized and boldfaced terms, e.g.:
image
vs. original:
image
@ronaldtse , please let me know if we should boldface the smart terms here, italicize non-smart terms, or keep the current formatting.
- Reference ISO 19101 has been added to bibliography since it was used in the original text.
- PDF: It seems that paragraphs aren't being separated in multi-paragraph notes to entries. @manuel489 , could you please confirm before I open a ticket to address this issue?

### CODE REVIEW

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: sources/document.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+= The Specification Model -- A Standard for Modular specifications
+:doctype: reference-model
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply sources/document.adoc]: I've now set them.

**Comment by ronaldtse** [comment]
> * Red text from the original document is highlighted in the generated one.

That's fine, thanks!

> * Smart terms were boldfaced in the original document. They are italicized in the generated one, i.e. usual markup for the smart terms is used. However, some other terms are boldfaced in the original document (and I kept the original formatting), so we now have a mixture of italicized and boldfaced terms, e.g.:
>   @ronaldtse , please let me know if we should boldface the smart terms here, italicize non-smart terms, or keep the current formatting.

This is correct -- let's keep the formatting relevant to the intended semantics, instead of presentation.

> * Reference ISO 19101 has been added to bibliography since it was used in the original text.

Thank you.

> * PDF: It seems that paragraphs aren't being separated in multi-paragraph notes to entries. @manuel489 , could you please confirm before I open a ticket to address this issue?

I can confirm this -- the paragraphs inside a NOTE are all mixed into one.

In HTML it is kept as separate paragraphs though.

Screenshot 2022-06-15 at 9 23 34 AM

And it is correct in Presentation XML:

```xml
        <termnote id="_3b974d8a-ecb8-0c82-efd5-bccd9eb48c4f">
          <name>Note 1 to entry</name>
          <p id="_86bc8221-980c-453d-bf8a-75df10228e0b">Each <em>requirement</em> is a normative criterion for a single <strong>type of standardization target</strong>. In this standard, requirements will be associated to
<em> conformance tests</em> that can be used to prove
compliance to the underlying criteria by the
<em>standardization target</em>.</p>
          <p id="_fd167ca6-1b18-3176-c9c4-66710aec076f">The implementation of a <em>requirement</em> is dependent on the type of
specification being written. A data specification requires data structures, but a
procedural specification requires software implementations. The view of a standard in
terms of a set of testable <em>requirements</em> allows us to
use set descriptions of both the standard and its implementations.</p>
          <p id="_cf2be0aa-724a-5cc8-9240-3c0d099d46ba">The specification of a <em>requirement</em> is usually expressed in terms
of a model of the <em>standardization target</em>, such as a UML model, or
an XML or SQL schema. Anything without a defined test is <em>a priori</em> not testable and
thus would be better expressed as a <em>recommendation</em>.</p>
          <p id="_29715bde-5e25-c1a4-0c9f-82f45544eff9"><em>Requirements</em> use normative language and in particular
are commands and use the imperative “shall” or similar imperative constructs.
Statements in standards which are not requirements and need to be either conditional
or future tense normally use “will” and should not be confused with requirements that
use “shall” imperatively.</p>
        </termnote>
```

@Intelligent2013 this is a bug -- can you please help handle it?

**Review by manuel489 — COMMENTED** [review]
> PDF: It seems that paragraphs aren't being separated in multi-paragraph notes to entries. @manuel489 , could you please confirm before I open a ticket to address this issue?

Confirmed.

**Comment by ronaldtse** [comment]
There is one more issue that some bibliographic entries from ISO are not rendering correctly. 

Screenshot 2022-06-15 at 10 02 29 AM

This is clearly a Metanorma issue as the formattedref is broken:

```adoc
* [[[iso9075,ISO/IEC 9075:2003]]], ISO/IEC JTC 1, ISO/IEC 9075:2003 -- Information Technology -- Database Languages - SQL.
```

```xml
  <bibitem id="iso9075" type="standard">
    <formattedref>
      /: ISO/IEC 9075:2003,
      <em>
      ISO/IEC JTC 1, ISO/IEC 9075:2003 — Information Technology — Database
      Languages — SQL.
    </em>
    . ISO, IEC (2003).
  </formattedref>
  <docidentifier type="metanorma-ordinal">
    [3]
  </docidentifier>
  <docidentifier type="ISO">
    ISO/IEC 9075:2003
  </docidentifier>
</bibitem>
```

@anermina I fixed syntax issues with some NOTEs which you can see here: b1832c5 . 

Merging now. Thank you for your help!

**Pull request merged**